src/app/entry/current-collections/current-collections.component.html

```diff
@@ -18,21 +18,30 @@
     <mat-card-title>Collections</mat-card-title>
   </mat-card-header>
   <mat-card-content>
-    <mat-list>
-      <mat-list-item
-        [matTooltip]="collectionOrganizations.organizationDisplayName"
-        *ngFor="let collectionOrganizations of currentCollections$ | async"
-      >
-        <a
-          [routerLink]="['/organizations', collectionOrganizations.organizationName, 'collections', collectionOrganizations.collectionName]"
-          routerLinkActive="router-link-active"
-          >{{ collectionOrganizations.collectionDisplayName }}</a
-        >
-      </mat-list-item>
-    </mat-list>
-    <div class="p-3">
+    <ng-container *ngFor="let collectionOrganization of currentCollections$ | async">
+      <div class="mb-1 single-spaced">
+        <ng-container *ngIf="collectionOrganization.organizationAvatarUrl; then withAvatar else withoutAvatar"></ng-container>
+        <ng-template #withAvatar>
+          <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="1rem">
+            <img class="tiny-org-logo" [src]="collectionOrganization.organizationAvatarUrl" alt="org logo" [appFallback]="orgLogoService.DEFAULT_URL"/>
+            <div class="pt-2 pb-2">
+              <ng-container *ngTemplateOutlet="collectionLink"></ng-container>
+            </div>
+          </div>
+        </ng-template>
+        <ng-template #withoutAvatar>
+          <div class="pt-2 pb-2">
+            <span class="weight-bold">{{ collectionOrganization.organizationDisplayName }}: </span>
```

Comment:
What happens with a long name? Does it wrap, truncate, truncate with ellipsis, or overflow? In any case, could you please attach a screenshot that includes an entry added to a logo-less org.

Reply:
Formatted like normal text, words wrap to next line.  Screenshot attached (I zeroed the Broad image before pulling up the page).